### PR TITLE
Aug 666

### PR DESCRIPTION
**Pull request by liorpaz**
Some comment here...
config for demo

### CODE REVIEW

**Comment by Vpougatsh** [comment]
Some comment here...
Other comment here...

File: handlers/slack_handler.py [hunk 1]
```diff
@@ -91,10 +87,12 @@ def get_list(self, user, channel, command, text):
 
     def handle_stats(self, user, channel, command, text):
         print "command stats"
-        self.sc.rtm_send_message(channel, "Fetching stats...")
+        # self.sc.rtm_send_message(channel, "Fetching stats...")
         channel_name, channel_type = self.get_channel_info(channel)
         res = channel_report(channel_name, channel_type)
         lines = ["{} by {} - {}".format(r[0].id, r[0].owner, r[1]["type"]) for r in res]
+        if not lines:
```
Comment (azk) [inline handlers/slack_handler.py]: so ugly!

File: config.py [hunk 2]
```diff
@@ -2,8 +2,8 @@
 from datetime import timedelta
 
 channels_config = [
-    ("code-reviews-hack", "pr"),
-    ("design-reviews-hack", "doc"),
+    ("code-reviews-demo", "pr"),
```
Comment (AssafMiz) [inline config.py]: blajshdis

File: config.py [hunk 3]
```diff
@@ -2,8 +2,8 @@
 from datetime import timedelta
 
 channels_config = [
-    ("code-reviews-hack", "pr"),
-    ("design-reviews-hack", "doc"),
+    ("code-reviews-deom", "pr"),
+    ("design-reviews-demo", "doc"),
```
Comment (rhosisey) [inline config.py]: demo not doem

File: handlers/slack_handler.py (hunk 1 above)
Comment (Vpougatsh) [reply handlers/slack_handler.py]: Any comment here...

File: config.py (hunk 2 above)
Comment (Vpougatsh) [reply config.py]: Any comment here...

Comment (OlgaMardechaev) [reply config.py]: more comments

**Pull request merged**